examples/cs/comm/tls/Tls.cs

```diff
@@ -13,7 +13,8 @@ namespace Bond.Examples.Tls
 
     public static class Tls
     {
-        static readonly IPEndPoint serviceEndpoint = new IPEndPoint(IPAddress.Loopback, EpoxyTransport.DefaultSecurePort);
+        static readonly IPEndPoint serviceEndpointIpv4 = new IPEndPoint(IPAddress.Loopback, EpoxyTransport.DefaultSecurePort);
+        static readonly IPEndPoint serviceEndpointIpv6 = new IPEndPoint(IPAddress.IPv6Loopback, EpoxyTransport.DefaultSecurePort);
```

Comment:
It doesn't matter that the same port number is being used? I'm guessing one of them is going to fail either way since localhost can only map to one so its a moot point?